README.md

```diff
@@ -200,6 +200,9 @@ var client = hdb.createClient({
 **Note** for MDC use cases: The system database and the target tenant database may be configured to work with different certificates.
 If so, make sure to include all the necessary TLS-related properties for both the databases in the client's options.
 
+In case you need custom logic to validate the server's host name against the certificate, you can pass a callback in the `checkServerIdentity` property, alongside other connection options. The callback is
```

Comment:
Minor comments in the readme:
- hostname - one word. 
- you can assign a callback function to the 'checkServerIdentity' property
maybe remove ' alongside other connection options.'